frontend/src/components/Forms/AdvancedContactSearchForm.js

```diff
@@ -11,6 +11,7 @@ import * as Strings from "@/constants/strings";
 const propTypes = {
   handleSubmit: PropTypes.func.isRequired,
   handleSearch: PropTypes.func.isRequired,
+  handleContactTypeChange: PropTypes.func.isRequired,
```

Comment:
Could you rename this to describe what it actually does? Something like `clearPartyNameFields` or `resetPartyNameFields`. I think it could be unnecessarily confusing to have a prop with this name and a method with this name in the same file. The method is more of the 'handler' than the prop.

Reply:
done